articles/logic-apps/create-managed-service-identity.md

```diff
@@ -189,7 +189,11 @@ for the resource you want to call.
 
 1. In the HTTP action, choose **Show advanced options**. 
 
-1. From the **Authentication** list, select **Managed Service Identity**, 
+1. From the **Authentication** list, select **Managed Service Identity**, fill in **URL** by the following URL :
```

Comment:
@cloudmelon, might I suggest this version, which might be clearer and also aligns the spacing for rendering the text correctly:

1. From the **Authentication** list, select **Managed Service Identity**. In the **URL** box, enter this URL, which makes the **Audience** box appear for you to set:

   ```url
   https://management.azure.com/subscriptions/<Azure-subscription-ID>?api-version-2016-06-01
   ```

Reply:
Hi @ecfan your suggestion looks good, I would like to precise ' you want to use ' by adding the endpoint that we can use in this case : https://docs.microsoft.com/en-gb/azure/active-directory/managed-identities-azure-resources/services-support-msi#azure-services-that-support-azure-ad-authentication
If you're keen on it, please feel free to proceed it. Thanks a lot :)

Reply:
@cloudmelon, just clarifying...are you suggesting that this doc include a reference to your suggested page that has supported authentication? @kevinlam1, what do you think?

Reply:
Right. I would propose the best way is to suggest the page contains supported authentification endpoint, and clarify that in our case default way is to use https://management.azure.com/ as URL.

Reply:
@cloudmelon, I edited the text based on this suggestion. Does this version look better now?

Reply:
@cloudmelon, if my suggested edits look good, please incorporate them into your branch. That way, the PR gets merged with you getting the credit as author. :) Thanks!

Reply:
Hi @ecfan updated as your suggestion. Thank you very much for your investment :)

Reply:
I reviewed it again and the text still does not line up with the HTTP action behavior:
"In the **URL** box, enter the endpoint URL for Azure Active Directory authentication..."
Here, do you mean the **Audience** box and not the **URL** box?
If you meant the **URI** box then (it should be URI not URL) and should be moved up to bullet 2.

"After you enter this URL, the Audience box appears for you to set that property..."
The Audience box appears after you select Managed Service Identity, not after you enter in this URL.

"For example, if you're using Azure Resource Manager, use this URL:"
In the previous sentence you are talking about the Audience and now you're talking about the URI box, which I found confusing. If you meant this to be under section 2 (and you remove the previous sentence about the timing of the appearance) then it makes more sense.